lib/pal/universal/WindowsRuntimeDeviceInformationImpl.cpp

```diff
@@ -120,8 +120,11 @@ namespace PAL_NS_BEGIN {
                         }
                     });
 
-                    token1 = ::Windows::System::Power::PowerManager::BatteryStatusChanged += onPowerSourceChanged;
-                    token2 = ::Windows::System::Power::PowerManager::PowerSupplyStatusChanged += onPowerSourceChanged;
+                    if (MAT::IsRunningInApp())
+                    {
+                        token1 = ::Windows::System::Power::PowerManager::BatteryStatusChanged += onPowerSourceChanged;
+                        token2 = ::Windows::System::Power::PowerManager::PowerSupplyStatusChanged += onPowerSourceChanged;
+                    }
```

Comment:
Why are these steps not required for unpackaged apps?

Reply:
Thanks for review and sorry for late reply as Chinese New Year holiday.

I didn't find any doc to say `BatteryStatusChanged` cannot work for unpackaged app, but it indeed crash in the deconstructor where we unsubscribe the event `::Windows::System::Power::PowerManager::BatteryStatusChanged -= token1;`

I removed these and it works fine.